tools/bin/hawq_ctl

```diff
@@ -111,10 +111,13 @@ class HawqInit:
             self.standby_host_name = ''
 
         if 'enable_secure_filesystem' in self.hawq_dict:
-            self.enable_secure_filesystem=self.hawq_dict['enable_secure_filesystem']
-            self.krb_server_keyfile =self.hawq_dict['krb_server_keyfile']
+            self.enable_secure_filesystem = self.hawq_dict['enable_secure_filesystem']
```

Comment:
**enable_secure_filesystem** could be set to **OFF** in hawq-site.xml. Current logic is assuming that security is on as long as enable_secure_filesystem parameter is present.